app/src/main/res/layout/fragment_personal_info.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".userProfileCreation.PersonalInfoFragment"
+    android:id="@+id/personalInfoFragment"
+    android:background="@drawable/image_mountain_background"
+    >
+
+    <LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+        xmlns:tools="http://schemas.android.com/tools"
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:orientation="vertical"
+        android:layout_gravity="center"
+        tools:context=".userProfileCreation.PersonalInfoFragment">
+        <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:padding="4dp"
+            android:text="@string/to_get_to_know_you_better"
+            android:textIsSelectable="false"
+            android:textSize="28sp"
+            android:textColor="@color/design_default_color_surface"
+            android:textAlignment="center"
+            android:textStyle="bold"/>
+        <EditText
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:textSize="30sp"
+            android:id="@+id/input_firstname_text_UserProfileCreation"
+            android:hint="@string/enter_your_firstname"
+            android:background="@color/design_default_color_surface"
+            />
```

Comment:
You could add these so that it stays on 1 line
`android:maxLines="1"`
`android:lines="1"`
`android:singleLine="true"`